Assignment/solution_student/index.html

```diff
@@ -15,17 +15,69 @@
     <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+      <div id="header">
+        <h1>Relaxr</h1>
+        <nav>
+          <ul>
+            <li><a href="#">About</a></li>
+            <li><a href="#">FAQ</a></li>
+            <li><a href="#">Team</a></li>
+            <li><a href="#">Contact Us</a></li>
+            <li><a href="#">Blog</a></li>
+          </ul>
+        </nav>
+      </div>
 
-      <main>
-        <p>Our HTML starter template!</p>
-      </main>
+      <div id="main">
+        <article>
+          <header>
```

Comment:
Not incorrect or troublesome, but since I can see you've got a pretty high comfort level for writing HTML already, and you've expressed interest in continuing, I'll give you some more detailed or nitpicky feedback :) So, I believe, that technically, the only time when the `<header>` element is _really_ needed, is if you have a group of header-like elements that need to be contained somehow, so if you had an `<h1>` _and_ some nav or some meta information about the post like date, author, etc.; _but_, if it's really just the `<h1>` there's really no need to wrap it in a container (unless there is for styling purposes), so you don't need the `<header>` here. Here's an article describing it for your reference: http://html5doctor.com/avoiding-common-html5-mistakes/#header-hgroup